docs/setup/schema.md

```diff
@@ -86,6 +86,10 @@
 }
 ```
 
+> [!Note]
+> If Clara DICOM Adapter is deployed via helm chart, find and modify `dicomPort` in `~/.clara/charts/dicom-adapter/templates/values.yaml` instead of modifying the scp port number in the `appsettings.json` file describe above.
```

Comment:
you could also give guidance on how to do this if helm install is being used i.e. : helm install ... -set dicom...=105 ... 
also you might want to note that since this is exposing a host port there may be failures if ports are already being used. 
So a common issues might be helpful to users.

Reply:
Created #44